cmd/microcloudd/main.go

```diff
@@ -141,6 +141,16 @@ func (c *cmdDaemon) Run(cmd *cobra.Command, args []string) error {
 
 		PreInitListenAddress: "[::]:" + strconv.FormatInt(service.CloudPort, 10),
 		Hooks: &state.Hooks{
+			PostBootstrap: func(ctx context.Context, state state.State, initConfig map[string]string) error {
+				addrPort, err := microTypes.ParseAddrPort(s.Address())
```

Comment:
Worth putting this in an anonymous function so that it can be used by both `PostBootstrap` and the `PostJoin` hook?

Reply:
Fixed.